index.html

```diff
@@ -145,23 +137,21 @@ <h3 class="subtitle">AATA</h3>
                             <li class="tag">Javascript</li>
                         </ul>
                     </div>
-                    <button class="btn" >See project</button>
+                    <button class="btn">See project</button>
                 </aside>
-            </div>
+            </div>   
         </section>
 
-        <section class="card right-top-radius">
-            <div class="project right-top-radius">
+        <section class="item about">
+            <div class="project">
                 <aside class="aside aside-1">
-                    <h1 class="blackTextColor">
+                    <h1>
                         About Myself
                     </h1>
-                    <p class="paragraph blackTextColor">
-                        I'm a software developer! I can help you build a product, feature or website look through some of my
-                        work and experience! If you like what you see and have a project you need coded, don't hestiate to
-                        contact me.
+                    <p class="paragraph">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
                     </p>
-                    <h5 class="paragraph purpleColor">LET'S CONNECT</h5>
+                    <h5>let's connect</h5>
```

Comment:
- [ ] I think you should ensure that the left aside of about section covers at least 30% of the body width. This will help us to ensure that the text does not over-stretch vertically. In addition, this is to ensure we stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) ☺️.

Your design
![image](https://user-images.githubusercontent.com/99394374/229755571-6913e5d4-055c-47c4-9072-df1da610336c.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/229755753-6469a76a-1d9c-473d-b355-da6763ebfd73.png)